### PR TITLE
Implement zoom in the highlighted area (pointer mode)

### PR DESCRIPTION
**Pull request by fnevgeny**
The action is bound to 'z' by default, exitState exits the mode. No consecutive
zooming is possible (though highlighting works), and drawing is not supported,
either.

### CODE REVIEW

File: demo/pdfpc-demo.tex [hunk 1]
```diff
@@ -222,6 +222,7 @@
           keys
         \item A rectangular area can be highlighted by using the drag mouse
           motion
+        \item Press \keys{Z} to zoom in the highlighted area; \keys{esc} to exit
```
Comment (AndreasBilke) [inline demo/pdfpc-demo.tex:225]: I think this should be a small `z`

**Comment by fnevgeny** [comment]
> I find it a little confusing that pressing `z` in zoom mode does not end the zoom

I had in mind to eventually implement everything correctly :) that is, in particular, allow for multiple consecutive zoom-in's - in which case `z` should be kept strictly for the zoom-in direction. Maybe later...

> and restore the previous view (with the highlighted region).

This may be desired indeed. Let me see...

**Comment by AndreasBilke** [comment]
Sorry, I didn't checked if this pattern was used before! Please revert (or
drop) the commit!

**Comment by AndreasBilke** [comment]
Yes! :-)

**Pull request merged**